sys/vm/swap_pager.c

```diff
@@ -2246,14 +2246,15 @@ swp_pager_meta_cheri_get_tags(vm_page_t page)
 	void * __capability *scan;
 	void * __capability *p;
 	struct swblk *sb;
-	vm_pindex_t swidx;
+	vm_pindex_t modpi, swidx;
 
 	swidx = rounddown(page->pindex, SWAP_META_PAGES);
```

Comment:
Might be worth renaming this to rdpi to match other places, or inlining it like other places do too?

Reply:
Yeah, inlining seems reasonable. Upstream would benefit from a helper function to do the lookup...